packages/web-components/src/components/filter-panel/filter-panel-modal.ts

```diff
@@ -61,6 +121,80 @@ class DDSFilterPanelModal extends HostListenerMixin(
     );
   }
 
+  get focusableElements() {
+    const { selectorCloseButton, selectorTabbable: selectorTabbableForModal } =
+      this.constructor as typeof DDSExpressiveModal;
+    return [
+      ...Array.from(
+        (this.shadowRoot?.querySelectorAll(
+          selectorCloseButton
+        ) as NodeListOf<HTMLElement>) || []
+      ),
+      ...Array.from(
+        this.querySelectorAll(
+          selectorTabbableForModal
+        ) as NodeListOf<HTMLElement>
+      ),
+    ];
+  }
+
+  private get _focusablePanelModalElements() {
```

Comment:
Instead of copy pasting this method, could we make the parent method protected or public so that we would inherit it here?